src/servidor/UsuarioService.ts

```
@@ -0,0 +1,18 @@
+import { BaseService } from '@/utils/BaseService.ts'
+import { Usuario } from '@/commons/interfaces.ts'
+
+class UsuarioService extends BaseService {
+  constructor() {
+    super('/usuarios')
+  }
+
+  async meuPerfil(): Promise<Usuario> {
+    return this.get<Usuario>("/meu-perfil")
```

ESLint

[error] Replace `"/meu-perfil"` with `'/meu-perfil'`

(prettier/prettier)